fuzz test the router VS a monolithic subgraph (#5302)
apollographql/router: fuzz/README.md

### Router

This target is sending the same query to a backend exposing all the types in a schema, and to the router started with a supergraph schema that points to that same backend for each subgraph, and checks that both answers are the same.
The invariant tested here is that however the schema is cut up, and however many subgraph requests it will take, the router will return the expected answer, as if it was just one API.

> Before launching it, you have to spawn the subgraph in `fuzz/suvbgraph` with `cargo run --release` and start a Router with the schema in `fuzz/subgrgraph/supergraph.graphql`. You can use whatever router configuration you need, allowing you to fuzz the router with specific features activated.

Reviewer: ````suggestion
Before launching the fuzz target, you have to spawn a subgraph server and start a router.

Start the subgraph server with this command:
```bash
cd fuzz/subgraph && cargo run --release
```

Start the router with a supergraph in `fuzz` directory and any router configuration you need to test. Below, the command uses `router.yaml` configuration in the `fuzz` directory:
```bash
 cargo run -- --config fuzz/router.yaml --supergraph fuzz/subgraph/supergraph.graphql --dev --log=trace
```
````

Run the fuzzer with this command:

```

> # Only works on Linux

Reviewer: ```suggestion
# Only works on Linux and MacOS
```
